charts/sonarqube/templates/networkpolicy.yaml

```diff
@@ -99,4 +99,7 @@ spec:
 {{- with .Values.networkPolicy.additionalNetworkPolicys -}}
 {{ toYaml . | nindent 2 }}
 {{- end }}
+{{- with .Values.networkPolicy.additionalNetworkPolicies -}}
```

Comment:
In this case, if we use both ways to add network policies, we end up with having the unsupported network policy and the support ones both added.

Shouldn't we "disable" `additionalNetworkPolicys` when `additionalNetworkPolicies` is used?